src/elements/core/styles/mixins/lists.scss

```diff
@@ -76,28 +76,119 @@
   }
 }
 
-@mixin step-list {
-  --sbb-step-list-marker-dimensions: #{functions.px-to-rem-build(34)};
-  --sbb-step-list-padding-block: var(--sbb-spacing-fixed-3x);
-  --sbb-step-list-padding-inline: var(--sbb-spacing-responsive-xxs);
-  --sbb-step-list-marker-to-text-gap: var(--sbb-spacing-responsive-xxxs);
-  --sbb-step-list-border-radius: var(--sbb-border-radius-4x);
+@mixin description-list {
+  // Support both top level usage (& is empty / falsy)
+  // and applied to a selector usage.
+  #{if(&, '&:where(dl)', 'dl')} {
+    @include typo.text-s--regular;
+
+    margin: 0;
+    padding: 0;
+    display: grid;
+
+    // Ensure that description is always attached to the label
+    grid-template-columns: auto minmax(20%, 1fr);
+    gap: var(--sbb-spacing-fixed-1x) var(--sbb-spacing-fixed-2x);
+    color: var(--sbb-color-iron);
+
+    :is(dt, dd) {
+      margin: 0;
+      padding: 0;
+    }
+  }
+}
+
+@mixin base-marker-list {
```

Comment:
I'd suggest creating SCSS vars for the CSS var names so that the output when consuming checkup-list and step-list is clean of `--sbb-list-marker-*` vars. and also re-declarations of a css var. The base-marker-list should be excluded from public api here: `src/elements/core/styles/_index.scss`